### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by arungalva**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/-feat--feature-request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Acceptance criteria**
+How can you verify that the feature actually works? ex.[If should be able to make a GET request to /greeeting and get back "hello world"
```
Comment (blessingomotayo) [inline .github/ISSUE_TEMPLATE/-feat--feature-request.md]: Please close the '[' bracket. Just a nitpick thing.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```
Comment (blessingomotayo) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe you an also add 'Enter the following...'. I know this is just a broad idea of what to add when creating an issue.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (blessingomotayo) [inline .github/ISSUE_TEMPLATE/bug_report.md]: i think this section should be more clear that it is meant to show either visually or by writing what the actual behaviour is compared to the expected behaviour.

File: .github/ISSUE_TEMPLATE/-feat--feature-request.md (hunk 1 above)
Comment (arungalva) [reply .github/ISSUE_TEMPLATE/-feat--feature-request.md]: 🤦‍♂️

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (arungalva) [reply .github/ISSUE_TEMPLATE/bug_report.md]: These are just examples. They will be able to enter anything

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (arungalva) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I could add section that says actual behavior but would that be similar to the description?

Comment (blessingomotayo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: From what I see, the description says 'describe the bug', but I'm asking to have a section that describes the actual behaviour of the bug like what does it affect. Basically actual vs expected sections.

Comment (arungalva) [reply .github/ISSUE_TEMPLATE/bug_report.md]: sure. I'll add it 👍🏾

**Comment by seanrt** [comment]
Let's go, good stuff

**Pull request merged**